Install path is /var/lib/roundcube for Debian. Don't try to guess it.

Index: roundcube/program/include/iniset.php
===================================================================
--- roundcube.orig/program/include/iniset.php	2013-04-14 11:57:27.972458142 +0200
+++ roundcube/program/include/iniset.php	2013-04-14 11:57:27.964457978 +0200
@@ -48,7 +48,7 @@
 define('RCMAIL_START', microtime(true));
 
 if (!defined('INSTALL_PATH')) {
-    define('INSTALL_PATH', dirname($_SERVER['SCRIPT_FILENAME']).'/');
+    define('INSTALL_PATH', '/var/lib/roundcube/');
 }
 
 if (!defined('RCMAIL_CONFIG_DIR')) {
